test/parallel/test-http-hostname-typechecking.js

```diff
@@ -21,7 +21,7 @@ vals.forEach((v) => {
 // These values are OK and should not throw synchronously
 ['', undefined, null].forEach((v) => {
   assert.doesNotThrow(() => {
-    http.request({hostname: v}).on('error', common.noop).end();
-    http.request({host: v}).on('error', common.noop).end();
+    http.request({hostname: v}).on('error', common.mustCall()).end();
+    http.request({host: v}).on('error', common.mustCall()).end();
```

Comment:
@addaleax My inclination would be to change `common.mustCall()` to `() => {}` to preserve the previous behavior. Perhaps bonus points for rethrowing the error if it's not `ECONNREFUSED`.